Pauli Twirling (PT) User Guide (#1848)
unitaryfund/mitiq: docs/source/guide/pt.md

# Pauli Twirling

> Pauli Twirling (PT) is an error mitigation technique in which,
> in the Mitiq implementation, particularly noisy operations (e.g. CZ and CNOT)

Reviewer: Suggest to simplify this wording a bit for readability.
```suggestion
In the Mitiq implementation of the Pauli Twirling (PT)  error mitigation technique,
particularly noisy operations (e.g. CZ and CNOT)
```
